.gitignore

```diff
@@ -1,4 +1,6 @@
 # Project exclude paths
```

Comment:
------------------------ This comment regards all modules. -----------------------------------------
First of all, place PaymentTransformer, PaymentProvider interfaces inside template module. 
The second, drop BluecodePayApplication
The third, add spring.factories file to bluecode module. You should place it to the resources directory inside META-INF folder. Use this as instruction: https://docs.spring.io/spring-boot/docs/2.0.0.M3/reference/html/boot-features-developing-auto-configuration.html
Add application class inside payment-backend and use bluecode module as a starter inside it.